lib/ecmarkdown.js

```diff
@@ -17,3 +18,9 @@ function parse(emd) {
   const p = new Parser(t);
   return p.parseDocument();
 }
+
+function parseFragment(emd) {
```

Comment:
Not sure what you mean? It seems equivalent to parse() and so should be a peer of it, wherever it is.